### DISCUSSION TITLE
Display Kannada repha by itself

### DISCUSSION THREAD

**Discussion by MayuraVerma** [harfbuzz/harfbuzz#2904] [Q&A] [answered]
In books and online content such as grammar, Kannada language study, to describe this symbol, it is desired to display the repha symbol by itself within quotes or even otherwise.

Is it possible display Kannada repha byself, with combination of any non display characters (ZWJ, ZWNJ, NBSP, etc)?

I do not see any mention of this in the unicode standard.

**Comment by dscorbett** [comment, marked as answer]
You can display the repha on a dotted circle with <0CB0, 0CCD, 25CC>, but there does not seem to be a standard way to display the repha by itself. I recommend you submit feedback to Unicode.

**Comment by MayuraVerma** [comment]
I am checking this in Chrome, Windows 10. 
We are not seeing repha instead Ra followed by virama and the dotted circle.
So, the reordering is not happening in presence of dotted circle.
![image](https://user-images.githubusercontent.com/14062593/111232285-d04b3b00-85b8-11eb-9ca0-18860cff5491.png)
